documentation/experimental_features.md

```diff
@@ -219,13 +219,18 @@ Futures will continue to execute in parallel. However, Bolt itself will not exit
 have completed in order to ensure that all work is finished. Any errors raised after the calling
 plan has finished will be logged at warn level for visibility.
 
-The `wait()` plan function accepts a single Future object or an array of Futures and blocks until
-they finish, with an optional timeout. If provided a timeout, any unfinished Futures will raise a
-timeout error if they have not completed within the timeout. You can return errors
-instead of raising them by passing `_catch_errors => true` to `wait()`. The `wait()`
-function returns the results from the Future blocks once they've all finished. If a
-Future errors, Bolt only raises the error after all other Futures finish executing and
-return to `wait()`.
+The `wait()` plan function accepts either a single Future object, an array of Futures, or no
+Futures. If not provided any Futures, the function will implicitly wait for all Futures created up
+to that point in the calling plan. Any Futures created in subplans or inside of `background()`
+blocks will not be waited on, even if the subplan is called inside a `background()` block. Users
+must call `wait()` in the subplan or `background()` block directly, or return Future objects to the
+calling plan to be explicitly passed to `wait()`.
+
+The function blocks until the Futures finish, with an optional timeout. If provided a timeout, any
+unfinished Futures will raise a timeout error if they have not completed within the timeout. You can
+return errors instead of raising them by passing `_catch_errors => true` to `wait()`. The `wait()`
+function returns the results from the Future blocks once they've all finished. If a Future errors,
+Bolt only raises the error after all other Futures finish executing and return to `wait()`.
```

Comment:
```suggestion
The function blocks until the Futures finish, with an optional timeout. If provided a timeout,
unfinished Futures raise a timeout error if they have not completed within the timeout. You can
return errors instead of raising them by passing `_catch_errors => true` to `wait()`. The `wait()`
function returns the results from the Future blocks once they've all finished. If a Future errors,
Bolt only raises the error after all other Futures finish executing and return to `wait()`.
```